docs/openapi/components/schemas/credentials/MasterBillOfLadingCertificate.yml

```diff
@@ -4,7 +4,7 @@ $linkedData:
 title: Master Bill of Lading Certificate
 description: >-
   A receipt for the cargo and a contract for transportation between a shipper and the ocean carrier. It may also be used as instrument of ownership (negotiable bill of lading) which can be bought, sold or traded while the goods are in transit. To be used in this manner, it must be a negotiable "order bill of lading". 
```

Comment:
```suggestion
  A receipt for the cargo and a contract for transportation between a shipper and the ocean carrier. It may also be used as instrument of ownership (negotiable bill of lading) which can be bought, sold, or traded while the goods are in transit. To be used in this manner, it must be a negotiable "order bill of lading". 
```